.github/actions/common/init-and-pull-gcc/action.yaml

```diff
@@ -28,7 +32,17 @@ runs:
           sleep 60
           git clone git://gcc.gnu.org/git/gcc.git
 
-      - name: Checkout master
+      - name: Detect unrecognized branches
+        shell: bash
+        if: ${{ inputs.branch != 'master' && inputs.branch != 'coordination' }}
+        run: |
+          echo "inputs.branch found \"${{ inputs.branch }}\""
+          exit 1
+
+      # Always checkout master branch. That way if init submodules cache uses the 
```

Comment:
I'm confused by this comment. Why should we checkout master if we're considering coordination branch?

Reply:
hmmm I was experimenting with things earlier and I thought I needed to be on a different branch. After experimenting more locally, it is unnecessary